docker/pravega/Dockerfile

```diff
@@ -9,6 +9,9 @@
 #
 FROM java:8
 
+RUN apt-get update && apt-get install -y -q rpcbind nfs-common \
```

Comment:
A `Dockerfile` is not a script that executes on the host machine. It is run during the build of a pravega release. Along with publishing the maven artifacts, we publish docker images to docker hub.

You can imagine that building a Dockerfile produces a binary that includes the OS, any dependencies that were installed as well as binaries (jars) that were added. Any host system that can install `docker`, regardless whether they are the same as the OS of the image, would be able to start the containers as such:

```
docker pull pravega/pravega:<VERSION>
docker run -d pravega/pravega:<VERSION> /opt/pravega/scripts/entrypoint.sh segmentstore
docker run -d pravega/pravega:<VERSION> /opt/pravega/scripts/entrypoint.sh controller
```

`docker-compose` is an abstraction over the commands above that ensures the containers are launched with proper configurations, etc.